cluster-scope/overlays/nerc-ocp-prod/kustomization.yaml

```diff
@@ -5,4 +5,6 @@ commonLabels:
 
 resources:
 - ../common
+- ../../bundles/external-secrets-clustersecretstore
```

Comment:
We do not actually want to set up a clustersecret store on ocp-prod, because this would potentially expose secrets to all cluster users. We're going to want to set up namespaced secret stores where we need them.

Reply:
...so if this were to simply pull in `../../bundles/external-secrets` instead it would be fine, and then we would add the namespaced resources in a separate PR.

Reply:
@chrisstafford can you take a look at this request?

Reply:
I'll take care of this

Reply:
@larsks did the implemented fix resolve your CR?